assets/js/modules/thank-with-google/components/settings/SettingsView.stories.js

```diff
@@ -52,28 +70,56 @@ function Template() {
 export const Default = Template.bind( null );
 Default.storyName = 'Default';
 Default.parameters = { features };
+Default.args = {
+	setupRegistry: ( registry ) => {
+		registry.dispatch( CORE_SITE ).receiveSiteInfo( {
+			postTypes: [
+				{ slug: 'posts', label: 'Posts' },
+				{ slug: 'pages', label: 'Pages' },
+				{ slug: 'custom', label: 'Custom' },
+			],
+			widgetsAdminURL: 'http://example.com/wp-admin/widgets.php',
+		} );
```

Comment:
This won't be needed once you add the default value for the `widgetsAdminURL` property in the `provideSiteInfo` function.